arcade/types.py

```diff
@@ -49,6 +52,22 @@
 
 RGBA255OrNormalized = Union[RGBA255, RGBANormalized]
 
+# equivalent to the following filters (under arcade.gl.enums):
+# NEAREST, LINEAR, NEAREST_MIPMAP_NEAREST, LINEAR_MIPMAP_NEAREST,
+# NEAREST_MIPMAP_LINEAR, LINEAR_MIPMAP_LINEAR
+OpenGlFilters: TypeAlias = Union[Literal[9728], Literal[9729], Literal[9984],
+                                 Literal[9985], Literal[9986], Literal[9987]]
+
+# equivalent to the following blend_functions (under arcade.gl.enums):
+# ZERO, ONE, SRC_COLOR, ONE_MINUS_SRC_COLOR,
+# SRC_ALPHA, ONE_MINUS_SRC_ALPHA, DST_ALPHA,
+# ONE_MINUS_DST_ALPHA, DST_COLOR, ONE_MINUS_DST_COLOR
+BlendFunctions: TypeAlias = Union[Literal[0], Literal[1], Literal[768],
```

Comment:
```suggestion
BlendFunction: TypeAlias = Union[Literal[0], Literal[1], Literal[768],
```
Same comment as above: since it represents a single piece of GL configuration, should it be singular?